packages/manager/apps/web/client/app/components/manager-order-catalog-price/manager-order-catalog-price.html

```diff
@@ -0,0 +1,57 @@
+<span data-ng-if="::$ctrl.price">
+    <span data-ng-if="::$ctrl.ASIA_FORMAT.includes($ctrl.ovhSubsidiary)">
```

Comment:
Maybe we could avoid to write logic in template :

```suggestion
    <span data-ng-if="::$ctrl.isAsiaFormat()">
```